### DISCUSSION TITLE
part-ii/2017-52/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#106] [Question comments]
Part II 2017 Dynamical Systems: Paper 3, Section II, A 

https://questions.tripos.org/part-ii/2017-52/

**Comment by eha1234** [comment]
I think when they say determine to $O(u^2)$, what they mean is find $u^2$ coefficients etc., so it's $Z(u, \mu) =$ stuff $+ O(u^3)$. Otherwise everything cancels and you get no information. 

The algebra was somewhat unpleasant so I'm highly suspicious of my answers but 

<details><summary>[spoiler]</summary>

$Z(u, \mu) = u^2+ O(u^2)$, $V(u,\mu) = - 4 u \mu + (\frac{1}{4}a- \frac{1}{2})u^3 + O(u^4)$, $\dot{u} = \frac{1}{2} u(\mu + (a-1)u^2)$, which gives you subcritical and supercritical pitchfork for $a>1$ and $a<1$ respectively. 
For higher orders, $Z(u, \mu) = u^2+u^4$, $\dot{u} = \frac{1}{2} u [\mu - \frac{3}{2} u^4]$
</details>

Comments and corrections appreciated!
